app/exporters/odt/cv.rb

```diff
@@ -78,29 +86,52 @@ def insert_personalien(report)
     # rubocop:enable Metrics/AbcSize
 
     def insert_competences(report)
-      competences_list = if include_core_competences_and_skills?
-                           [core_competences_list, competence_notes_list].flatten
-                         else
-                           [competence_notes_list].flatten
-                         end
-      report.add_table('COMPETENCES', competences_list, header: true) do |t|
+      interest_skills(report) if include_skills_by_interests?
+      core_competences(report)
+    end
+
+    def interest_skills(report)
+      interest_value = skill_interest_value
```

Comment:
no need to set it here when you use it in the next line.
just do `interest_skills_list = skills_by_interest_value(skill_interest_value)` on the next line